test/regress/cli/run_regression.py

```diff
@@ -251,6 +251,82 @@ def run_internal(self, benchmark_info):
             print_ok("OK")
         return exit_code
 
+class AletheTester(Tester):
+    def __init__(self):
+        super().__init__("alethe")
+
+    def applies(self, benchmark_info):
+        logic = ""
+
+        for line in benchmark_info.benchmark_content.split("\n"):
+            if "(set-logic" in line:
+                logic = line.strip()
+
+        # (set-logic ...) not appear
+        if logic == "":
+            return False
+
+        logic = logic.split(" ")[1][0:-1]
+
+        return (
+            benchmark_info.benchmark_ext != ".sy"
+            and benchmark_info.expected_output.strip() == "unsat"
+        )
+
+    def run_internal(self, benchmark_info):
+        exit_code = EXIT_OK
+
+        with tempfile.NamedTemporaryFile(suffix=".smt2.proof") as tmpf:
+            cvc5_args = benchmark_info.command_line_args + [
+                "--dump-proofs",
+                "--proof-format=alethe",
+                "--proof-granularity=theory-rewrite"
```

Comment:
this option is no longer necessary since setting the format for Alethe automatically sets the granularity to this level.